src/api/uploads/MultiputUpload.js

```diff
@@ -205,8 +205,9 @@ class MultiputUpload extends BaseMultiput {
             }
 
             if (
-                (errorData && (errorData.status === 403 && errorData.code === 'storage_limit_exceeded')) ||
-                (errorData.status === 403 && errorData.code === 'access_denied_insufficient_permissions')
+                (errorData &&
+                    (errorData.status === UNAUTHORIZED_CODE && errorData.code === 'storage_limit_exceeded')) ||
```

Comment:
If we're starting the constants game can you add the strings as well? :)

Reply:
Lets create a separate ticket for that